src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String COMMA = ",";
+    private static final String OUTPUT_FORMAT = "%s,%d\n%s,%d\nresult,%d";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] totals;
+        try {
+            totals = readAndProcessInput(fromFileName);
+            writeOutput(toFileName, totals);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file: " + fromFileName, e);
+        }
```

Comment:
![image](https://github.com/mate-academy/jv-work-with-file/assets/20326858/bb95b79f-8598-41af-8759-f945b3f941e3)
readAndProcessInput - when you have **And** in the method name it shows that this method is not responsible for **one** task.
```
String dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeToFile(report, toFileName);
```